src/test/java/core/basesyntax/service/WriterTest.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax.service;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.FileNotFoundException;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class WriterTest {
+
+    private static final String WRITER_RESULT_PATH = "src/test/java/core/basesyntax/resources/writerResult.txt";
+    private static final String WRONG_PATH = "src/test/java/core/basesyntax/resources";
+
+    @Test
+    void writeToFile_emptyReportProvided() {
+        Writer.writeToFile(WRITER_RESULT_PATH, "");
+
+        int actualLinesQuantity = Reader.readLines(WRITER_RESULT_PATH).size();
+        int expectedLinesQuantity = 0;
+
+        assertEquals(expectedLinesQuantity, actualLinesQuantity);
+
```

Comment:
Optional: Please remove unnecessary empty lines 23 and 50

Reply:
Thanks, have done.